docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -151,6 +151,65 @@ to perform the remaining query processing on a smaller amount of data.
 The specifics for the supported pushdown of table joins varies for each data
 source, and therefore for each connector.
 
+However, there are some generic conditions that must be met in order for a join
+to be pushed down:
+
+* join pushdown must be enabled
+* all predicates that are part of the join must be possible to be pushed down
+* none of the tables in the join must be empty
```

Comment:
```suggestion
```

where does this come from?
i am not aware of this condition

Reply:
You are correct. I was looking at the wrong code for this condition.